miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultDeliverableService.java

```diff
@@ -0,0 +1,76 @@
+package uk.ac.bbsrc.tgac.miso.service.impl;
+
+import java.io.IOException;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.transaction.support.TransactionTemplate;
+
+import uk.ac.bbsrc.tgac.miso.core.data.impl.Deliverable;
+import uk.ac.bbsrc.tgac.miso.core.security.AuthorizationManager;
+import uk.ac.bbsrc.tgac.miso.core.service.DeliverableService;
+import uk.ac.bbsrc.tgac.miso.core.service.exception.ValidationError;
+import uk.ac.bbsrc.tgac.miso.core.store.DeletionStore;
+import uk.ac.bbsrc.tgac.miso.persistence.DeliverableDao;
+import uk.ac.bbsrc.tgac.miso.persistence.SaveDao;
+import uk.ac.bbsrc.tgac.miso.service.AbstractSaveService;
+
+@Service
+@Transactional(rollbackFor = Exception.class)
+public class DefaultDeliverableService extends AbstractSaveService<Deliverable> implements DeliverableService {
```

Comment:
* Override `validateDelete` to check for usage
* Override `authorizeUpdate` to prevent non-admins from creating/editing